test/LibYear.Lib.Tests/PackageVersionCheckerTests.cs

```diff
@@ -80,18 +103,18 @@ public async Task LatestDoesNotIncludePrerelease()
         public async Task LatestDoesNotIncludeUnpublished()
         {
             //arrange
-            var metadata = PackageSearchMetadataBuilder.FromIdentity(new PackageIdentity("test", new NuGetVersion(1, 2, 3))).Build();
+            var metadata = PackageSearchMetadataBuilder.FromIdentity(new PackageIdentity("test", new PackageVersion(1, 2, 3))).Build();
             var metadataResource = Substitute.For<PackageMetadataResource>();
             metadataResource.GetMetadataAsync(Arg.Any<string>(), Arg.Any<bool>(), Arg.Any<bool>(), Arg.Any<SourceCacheContext>(), Arg.Any<ILogger>(), Arg.Any<CancellationToken>())
                 .Returns(m => new List<IPackageSearchMetadata> { metadata }, m => throw new Exception(":("));
 
-            var v1 = new VersionInfo(new NuGetVersion(1, 2, 3), new DateTime(2015, 1, 1));
-            var v2 = new VersionInfo(new NuGetVersion(2, 3, 4), new DateTime(1900, 1, 1));
-            var versionCache = new Dictionary<string, IList<VersionInfo>> { { "test", new List<VersionInfo> { v1, v2 } } };
+            var v1 = new Release(new PackageVersion(1, 2, 3), new DateTime(2015, 1, 1));
+            var v2 = new Release(new PackageVersion(2, 3, 4), new DateTime(1900, 1, 1), false);
```

Comment:
OK, I see it now -- the NuGet API previously returned a date of 1/1/1900 for "unpublished" rather than the separate `IsListed` prop -- can we update this line of the test so 2.3.4 is released after 1.2.3 (others have it as 1/1/2016) so that we can be more confident that the logic is checking against the new `bool` flag

Reply:
Oh, right.  I meant to make that change when I updated this test and apparently forgot.  It's fixed now.